File: CoreJava/interface-polymorphism/src/polymorphismcomponets/Boy.java
package polymorphismcomponets;

public class Boy implements IMannerable, IEmotionable {

	@Override
	public void cry() {
		System.out.println("Boy Cry.");
	}

	@Override
	public void laugh() {
		System.out.println("Boy laugh.");
	}

	@Override
	public void wish() {
		System.out.println("Boy wish.");
	}

	@Override
	public void depart() {
		System.out.println("Boy depart.");
	}

}

File: C#/OOP/InterfacePolymorphism_App/InterfacePolymorphism_App/Boy.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace InterfacePolymorphism_App
{
    class Boy : IEmotionable
    {
        public void cry()
        {
            Console.WriteLine("Boy is Crying");
        }

        public void laugh()
        {
            Console.WriteLine("Boy is Laughing");
        }
    }
}
